packages/react/src/components/hero/Hero.stories.tsx

```diff
@@ -0,0 +1,546 @@
+      <Navigation.Item href="#" label="Link" onClick={(e) => e.preventDefault()} />
+      <Navigation.Item href="#" label="Link" onClick={(e) => e.preventDefault()} />
+      <Navigation.Dropdown label="Dropdown">
+        <Navigation.Item href="#" label="Link" onClick={(e) => e.preventDefault()} />
+        <Navigation.Item href="#" label="Link" onClick={(e) => e.preventDefault()} />
+        <Navigation.Item href="#" label="Link" onClick={(e) => e.preventDefault()} />
+        <Navigation.Item href="#" label="Link" onClick={(e) => e.preventDefault()} />
+      </Navigation.Dropdown>
+    </Navigation.Row>
+
+    {/* NAVIGATION ACTIONS */}
+    <Navigation.Actions>
+      {/* LANGUAGE SELECTOR */}
+      <Navigation.LanguageSelector label="FI">
+        <Navigation.Item href="#" onClick={(e) => e.preventDefault()} lang="fi" label="Suomeksi" />
+        <Navigation.Item href="#" onClick={(e) => e.preventDefault()} lang="sv" label="På svenska" />
+        <Navigation.Item href="#" onClick={(e) => e.preventDefault()} lang="en" label="In English" />
+        <Navigation.Item href="#" onClick={(e) => e.preventDefault()} lang="fr" label="En français" />
+        <Navigation.Item href="#" onClick={(e) => e.preventDefault()} lang="de" label="Auf deutsch" />
+        <Navigation.Item href="#" onClick={(e) => e.preventDefault()} lang="ru" label="По-русски" />
+      </Navigation.LanguageSelector>
+    </Navigation.Actions>
+  </Navigation>
+);
+
+export const ImageLeft = (args) => {
+  const heroProps: HeroProps = {
+    koros: args.koros,
+    centeredContent: args.centeredContent,
+    theme: { ...imageLeftOrRightTheme, ...args.theme },
+    imageSrc: args.imageSrc || imageFile,
+    variant: 'imageLeft',
+  };
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent buttonTheme="black" />
+    </Hero>
+  );
+};
+ImageLeft.argTypes = {
+  ...getDisabledControl('variant'),
+};
+
+export const ImageRight = (args) => {
+  const heroProps: HeroProps = {
+    koros: args.koros,
+    centeredContent: args.centeredContent,
+    theme: { ...imageLeftOrRightTheme, ...args.theme },
+    imageSrc: args.imageSrc || imageFile,
+    variant: 'imageRight',
+  };
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent buttonTheme="black" />
+    </Hero>
+  );
+};
+ImageRight.argTypes = {
+  ...getDisabledControl('variant'),
+};
+
+export const WithoutImage = (args) => {
+  const heroProps: HeroProps = {
+    theme: {
+      '--background-color': '#9fc9eb',
+      '--color': '#000',
+      '--koros-color': '#009246',
+      '--koros-height': '82px',
+      ...args.theme,
+    },
+    koros: { type: 'pulse', ...args.koros },
+    centeredContent: args.centeredContent !== false,
+    variant: 'noImage',
+  };
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent />
+    </Hero>
+  );
+};
+
+WithoutImage.argTypes = {
+  ...getDisabledControl('variant'),
+  ...getDisabledControl('imageSrc'),
+};
+
+export const WithoutImage2 = (args) => {
```

Comment:
I tried to came up with something better, but story names cannot (there is no much space) be very long. Descriptive name would be "No image, bottom koros overlaying space below" 😄  Heroes are pretty much just visual sugar, so the story name does not say much, but devs can see the component and see what the story is about.

The story is a good one, because it shows the settings for that version of that variant.  There are three different kind of non-image heroes created with `{ variant: "noImage" }`

There is a playground for theme at the moment. Is it wrongly built? It is good to have all args in the story props, because koros and theme settings work hand in hand.